railties/lib/rails/test_unit/testing.rake

```diff
@@ -29,31 +29,35 @@ namespace :test do
     success || exit(false)
   end
 
+  def test_paths(paths)
```

Comment:
Given the method name is `rake_run`, what do you think about naming it like this:

```suggestion
  def run_rake_tasks(paths)
```
Or maybe:

```suggestion
  def run_test_tasks(paths)
```

Reply:
hey! thanks for the feedback

couldn't that be confusing? I mean, if I read `run_test_tasks` I would expect the argument to be the tasks I want to run (`:models`, `:system`, etc).

Maybe `run_tests_in_paths`? so from the name you know the argument will be an array of paths and it will run the tests inside those paths.

I'm ok using `run_test_tasks` though, it's my first contribution so I prefer to follow the advice of more experienced contributors

Reply:
I actually like `run_tests_in_paths`.